be/src/storage/lake/transactions.h

```diff
@@ -60,7 +60,8 @@ StatusOr<TabletMetadataPtr> publish_version(TabletManager* tablet_mgr, int64_t t
 //
 // Return:
 // - Returns OK if the copy was successful, asynchronous deletion does not affect the return value.
-Status publish_log_version(TabletManager* tablet_mgr, int64_t tablet_id, int64_t txn_id, int64_t log_version);
+Status publish_log_version(TabletManager* tablet_mgr, int64_t tablet_id, const int64_t* txn_ids,
+                           const int64_t* log_versions, int txns_size);
```

Comment:
Consistent with the functions above use `std::span`?

Reply:
Unfortunately, after using std::span, cherry-pick to branch-3.1 had to change back to using arrays.